.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "go.inferGopath": false
```

Comment:
what's this?

Reply:
Apparently it was automatically added by vscode, and this file is not listed in gitignore, should I remove it?

the configuration itself just tells vscode if the gopath should be inferred or not

Reply:
I'd remove it, just in case